src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -41,3 +62,25 @@ def get(
         self, request: Request, project: Project, release_threshold: ReleaseThreshold
     ) -> HttpResponse:
         return Response(serialize(release_threshold, request.user), status=200)
+
+    def put(
+        self, request: Request, project: Project, release_threshold: ReleaseThreshold
+    ) -> HttpResponse:
+        serializer = ReleaseThresholdPUTSerializer(data=request.data)
+
+        if serializer.is_valid():
+            result = serializer.validated_data
+            kwargs = {}
+            if "threshold_type" in result:
+                kwargs["threshold_type"] = result["threshold_type"]
+            if "trigger_type" in result:
+                kwargs["trigger_type"] = result["trigger_type"]
+            if "value" in result:
+                kwargs["value"] = result["value"]
+            if "window_in_seconds" in result:
+                kwargs["window_in_seconds"] = result["window_in_seconds"]
+
+            if kwargs:
+                release_threshold.update(**kwargs)
```

Comment:
I'm a little confused by `kwargs` here, could you not pass result directly into the update? I think any random fields not in the serializer won't be included in `validated_data`

Reply:
hmmm...
actually you're right, that should be fine